app/models/timer.rb

```diff
@@ -0,0 +1,31 @@
+class Timer < ActiveRecord::Base
+
+  belongs_to :player
+  belongs_to :game
+
+  def initialize
+    @start_time = nil
+    @stop_time = nil
+    @running = false
+    @time_left = 500
+  end
+
+  def start
+    @time_left 
```

Comment:
I made changes but won't push yet since I'd like to solve these current failing tests.